src/PKSim.Infrastructure/InfrastructureRegister.cs

```diff
@@ -83,10 +83,11 @@ private static void registerRunOptionsIn(IContainer container)
          container.Register<StartOptions, IStartOptions, StartOptions>(LifeStyle.Singleton);
       }
 
-      private static IContainer initializeContainer()
+      private static IContainer initializeContainer(bool registerContainerAsStatic)
       {
          var container = new CastleWindsorContainer();
-         IoC.InitializeWith(container);
+         if (registerContainerAsStatic) 
```

Comment:
static stuff registered for PKSim starrt (defualt) .But false for R so that we are not relying on any IoC stuff